config.json

```diff
@@ -1,4 +1,4 @@
 {
-    "default_hs_url": "https://matrix.org",
-    "default_is_url": "https://vector.im"
+    "default_hs_url": "http://localhost:8008",
+    "default_is_url": "http://localhost:8090"
```

Comment:
no thanks... plus you should never need to do this hack nowadays given it's persisted in localstorage